### PR TITLE
use new dropshot tag config to enumerate tags

### PR DESCRIPTION
**Pull request by ahl**
This will require additional work in terms of the specific tags and their descriptions. I expect the external docs link will point to our product documentation.

### CODE REVIEW

**Comment by jessfraz** [comment]
wait does this generate the openapi spec tho? with `tags`

**Comment by ahl** [comment]
oy. it should. one second..

File: openapi/nexus.json [hunk 1]
```diff
@@ -6085,5 +6085,126 @@
...
+      "description": "Firewall operation controls the flow of network data into a VPC",
+      "externalDocs": {
+        "url": "http://oxide.computer/docs/#xxx"
+      }
+    },
+    {
+      "name": "hidden",
+      "description": "TODO operations that will not ship to customers",
+      "externalDocs": {
+        "url": "http://oxide.computer/docs/#xxx"
```
Comment (jessfraz) [inline openapi/nexus.json:6108]: These are "https://docs.oxide.computer/api/{lowercase tag name}" if you want to fill them in with the real url

Comment (jessfraz) [reply openapi/nexus.json:6108]: But I don't technnically need that right now anyways but just so you know

Comment (ahl) [reply openapi/nexus.json:6108]: These aren't links to the API documentation; they're links to the product documentation.

Comment (jessfraz) [reply openapi/nexus.json:6108]: Ah ha, I see, like a tutorial? `https://docs.oxide.computer/tutorials/{thing}` ?

Comment (jessfraz) [reply openapi/nexus.json:6108]: or the thing in the rack? regarless I gues it doesn't really matter

Comment (ahl) [reply openapi/nexus.json:6108]: Yeah, like that. e.g. https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instances-and-amis.html

Comment (jessfraz) [reply openapi/nexus.json:6108]: ah gotcha!

**Pull request merged**